docs/source/troubleshooting.rst

```diff
@@ -24,13 +24,13 @@ Viewing website
 Images are not displayed when I view the website
```

Comment:
well, if you are talking about embedded images or icons, the converter will spaz and raise an exception all by itself and tell you to check the file exists/is named correctly/is in the right place, does this need to be in the faq?...